packages/components/src/components/breadcrumb-item/breadcrumb-item.custom.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from 'lit';
+
+export default css`
+  .breadcrumb-item {
+    color: var(--syn-color-neutral-500);
+    font-size: var(--syn-font-size-x-small);
+    font-weight: var(--syn-font-weight-normal);
+  }
+
+  /**
+   * Make sure to remove the border radius.
+   * Without this, focus will be rounded
+   */
+  .breadcrumb-item__label {
+    border-radius: var(--syn-border-radius-none);
```

Comment:
Focus from shoelace and the focus of the current figma version differ: 
Figma: 
![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/fcef20ff-179d-4332-929d-4a78afbf87f9)

Shoelace / current dev version:
![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/16e66307-4635-4f82-b39a-2852574fbbdb)

Reply:
This is unfortunately not possible to to, because Shoelace does not add the prefix- or suffix-slots to the surrounding link or button element. It would also not work when using `syn-dropdown` as slot content, as in this case everything would be need to be part of the dropdown.